src/robot_description/models/master_ASM/urdf/master_ASM.urdf

```diff
@@ -140,14 +141,6 @@ Stephen Brawner (brawner@gmail.com)
       <topic>/depth/image</topic>
       <info_topic>/depth/camera_info</info_topic>
     </sensor>
-    <sensor name="imu_sensor" type="imu">
```

Comment:
if you are going to remove the imu sensor, then remove it from the ros gz bridge too please